source/docs/03-metadata.xml

```diff
@@ -1677,6 +1677,53 @@
                         <specDesc key="treatSched"/>
                      </specList>
                   </p>
+                  <p>
+                     <specList>
+                        <specDesc key="acquisition"/>
+                        <specDesc key="provenance"/>
+                     </specList>
+                  </p>
+                  <p><gi scheme="MEI">acquisition</gi> is a container for recording the process of the acquirement of an item by the holding institution. In comparison, <gi scheme="MEI">provenance</gi> deals with the history of ownership or custodianship of an item. Both elements allow for the choice of either text or more structured information when formulating the specific encoding. It is recommended to make use of <gi scheme="MEI">p</gi> elements when a text-centred encoding is favoured and to use the <gi scheme="MEI">eventList</gi> element for a more structured encoding. It is up to the encoder to decide where the information is most appropriate for the particular project or encoding purposes.</p>
+                  <p>
+                     <specList>
+                        <specDesc key="exhibHist"/>
+                     </specList>
+                  </p>
+                  <p>The <gi scheme="MEI">exhibHist</gi> element is intended to contain descriptions of one or more public exhibitions of a bibliographic item. Often exhibitions include an additional description in the form of a tag for the public that accompanies the item on display. These descriptions may even be printed in a published exhibition catalogue, so the encoding may also include information about why the object was shown or what was significant about the exhibition. When formulating the encoding, it is at the discretion of the encoder whether to opt for text or more structured information. Text-centred or essayistic encoding is made possible by <gi scheme="MEI">p</gi> elements in <gi scheme="MEI">exhibHist</gi>, among others. For more structured encodings, it is recommended to use the <gi scheme="MEI">eventList</gi> element contained in <gi scheme="MEI">exhibHist</gi>. Within MEI <gi scheme="MEI">exhibHist</gi> is conceptually bound to the item-level, however, as a child element to <gi scheme="MEI">history</gi>, <gi scheme="MEI">exhibHist</gi> is allowed in several places. It is up to the encoder to decide where the information is most appropriate for the particular project or encoding purposes. As best practice, it is recommended that <gi scheme="MEI">exhibHist</gi> is encoded as a child element of either <gi scheme="MEI">item</gi> or <gi scheme="MEI">physLoc</gi>.</p>
```

Comment:
```suggestion
                  <p>The <gi scheme="MEI">exhibHist</gi> element is intended to contain descriptions of one or more public exhibitions of a bibliographic item. Often exhibitions include an additional description in the form of a tag for the public that accompanies the item on display. These descriptions may even be printed in a published exhibition catalogue, so the encoding may also include information about why the object was shown or what was significant about the exhibition. When formulating the encoding, it is at the discretion of the encoder whether to opt for text or more structured information. Text-centred or essayistic encoding is made possible by <gi scheme="MEI">p</gi> elements in <gi scheme="MEI">exhibHist</gi>, among others. For more structured encodings, it is recommended to use the <gi scheme="MEI">eventList</gi> element contained in <gi scheme="MEI">exhibHist</gi>. Within MEI <gi scheme="MEI">exhibHist</gi> is conceptually bound to the item-level. As an element it is thus not permitted at the work or expression level and only permitted at the manifestation level, if the manifestation comprises a manifestation singleton.</p>
```

Reply:
The text was written before the schematron rule for `<history>` was adapted as to constrain the use of `<exhibHist>` et. al.
Thus I propose minor changes to the text.